code/processes/wdb.q

```diff
@@ -240,21 +241,32 @@ setcompression:{[compression] if[3=count compression;
 				]}
 resetcompression:{setcompression 16 0 0 }
 
+movetohdb:{[dw;hw;pt]
```

Comment:
I think a few more comments in this function to say what it is trying to do would be good